### DISCUSSION TITLE
Welcome to global-workflow Discussions!

### DISCUSSION THREAD

**Discussion by KateFriedman-NOAA** [NOAA-EMC/global-workflow#207] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andytangborn** [comment]
I'm setting up a coupled aerosol experiment, and I think it's not quite right yet. In the config.base file (after running setup_expt.py) three lines look suspicous:

export FDATE=@FDATE@
export NMEM_ENKF=@NMEM_ENKF@
export CASE_ENKF="@CASEENS@"

If they are incorrect, where do I make changes to fix them in the setup.  


And I hope this is the right place to ask questions like this, let me know if this should go someplace else.

**Reply by WalterKolczynski-NOAA** [reply]
Those three lines are only substituted in when running a cycled experiment, so that isn't out of the ordinary (actually, we're not using FDATE at all right now). They never get used by the forecast-only system.

**Reply by WalterKolczynski-NOAA** [reply]
Also, this should be in its own discussion thread.

**Comment by andytangborn** [comment]
Ok. Thanks. This is helpful to know.

**Comment by mjmurph** [comment]
Hi all,  I am Michael a researcher at UCSDD who is working with FV3-GFS for JEDI experiments.  I was able to run the workflow successfully on our machine for a test run using saved input files in forecast only mode (no DA).  Now I want to run the same experiment with IAU turned off.  I changed all the DOIAU variables to "NO" in the config files but I get errors about file not found when I take away the increment files and run.  Perhaps I took away too many input files?  Which ones are required only for IAU and can be removed after turning it off?  Any advice will be appreciated.

**Comment by MarcusGDaniels** [comment]
Hi,

I'm trying to setup initial conditions for GFSv16.3 but the files that are available on https://nomads.ncep.noaa.gov/pub/data/nccf/com/gfs/v16.3/ are all .nc, whereas the scripts like ush/global_chgres_driver.sh assume the files are .nemsio.   Do I need older files to do this?  Where could I get them?  Or is there a way to bulk convert from netcdf back to nemsio?

Thanks,

Marcus

**Reply by WalterKolczynski-NOAA** [reply]
The workflow should automatically chooses the correct filetype. If you encounter issues, please open a new discussion and we will try to help you (but possibly not until the New Year, as most of us are on leave).

**Comment by WalterKolczynski-NOAA** [comment]
Please open a new thread for any new discussions. Locked.
